src/main/webapp/css/srp_style.css

```diff
@@ -98,3 +102,17 @@ pre::-webkit-scrollbar {
   margin-right: 5px;
   vertical-align: bottom;
 }
+
+.vote {
```

Comment:
Can you put this above  .content so the CSS order somewhat follows the order of appearance?